tests/test_result_to_dict.py

```diff
@@ -35,6 +35,17 @@ class Data(Base):
     data_enum = Column(Enum(MyEnum))
 
 
+def convert_str_to_dict(result):
+    obj = result.get('object')
+    if obj and isinstance(obj, str):
+        result['object'] = json.loads(obj)
+        return result
+    return None

```

Comment:
if we have callback that returns None. there is no need to return None again. 
`return result`